add-ons/capsule/values.yaml

```diff
@@ -3,12 +3,12 @@ capsule:
     release: capsule
   # Make sure cert-manager is installed to generate certificates for webhooks automatically.
   certManager:
-    generateCertificates: true
+    generateCertificates: false
   tls:
     # Disable automatic TLS certificate creation by the chart itself.
-    create: false
+    create: true
     # Delegates TLS certificate management to cert-manager. Ensure cert-manager is properly set up.
```

Comment:
Set to false to delegate TLS certificate management to cert-manager. Ensure cert-manager is appropriately set up.